spec/index.html

```diff
@@ -324,6 +324,9 @@ <h3>Triple Terms and Reification</h3>
       For a <a>triple term</a> to be asserted,
       it must also appear in a graph as an <a>asserted triple</a>.</p>
```

Comment:
I think @gkellogg makes [a good point](https://github.com/w3c/rdf-concepts/pull/219#issuecomment-3062916373). To address it, perhaps the following:
```suggestion
       For a <a>triple term</a> to be asserted in a graph,
       it must appear as both
       an <a>asserted triple</a>
       and a <a>triple term</a>
       in that same graph.</p>
```

Reply:
The proposed language here is a bit redundant, I think. It could just be the following:

```suggestion
      For a <a>triple term</a> to be asserted in a graph,
      it must appear as
      an <a>asserted triple</a>
      in that same graph.</p>
```

Reply:
I like @gkellogg's suggestion better. I also notice that the original text used "also" ("must also appear"), which i would rather keep...